### PR TITLE
✨ Update BQ table name for raw results

### PR DESCRIPTION
**Pull request by laurentsimon**
Update BQ table name for raw result's releasetest
no breaking changes
```release-notes
Update BQ table name for raw result's releasetest
```

### CODE REVIEW

File: cron/k8s/transfer.release-raw.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ spec:
             - name: RAW_SCORECARD_DATA_BUCKET_URL
               value: "gs://ossf-scorecard-rawdata-releasetest"
             - name: RAW_SCORECARD_BIGQUERY_TABLE
-              value: "scorecard_raw-releasetest"
+              value: "scorecard_rawdata-releasetest"
```
Comment (azeemshaikh38) [inline cron/k8s/transfer.release-raw.yaml:36]: If BQ table is not already created, maybe rename to `scorecard-rawdata-releasetest` to maintain convention? Feel free to ignore.

**Review by azeemshaikh38 — APPROVED** [review]

**Comment by laurentsimon** [comment]
this may be a one off:
```
protoc --go_out=../../../ cron/data/request.proto
protoc-gen-go: program not found or is not executable
Please specify a program using absolute path or make sure the program is available in your PATH system variable
--go_out: protoc-gen-go: Plugin failed with status code 1.
make: *** [Makefile:10[8](https://github.com/ossf/scorecard/runs/5633929514?check_suite_focus=true#step:8:8): cron/data/request.pb.go] Error 1
```

fyi @naveensrinivasan @azeemshaikh38

**Comment by azeemshaikh38** [comment]
> this may be a one off:
> 
> ```
> protoc --go_out=../../../ cron/data/request.proto
> protoc-gen-go: program not found or is not executable
> Please specify a program using absolute path or make sure the program is available in your PATH system variable
> --go_out: protoc-gen-go: Plugin failed with status code 1.
> make: *** [Makefile:10[8](https://github.com/ossf/scorecard/runs/5633929514?check_suite_focus=true#step:8:8): cron/data/request.pb.go] Error 1
> ```
> 
> fyi @naveensrinivasan @azeemshaikh38

It is a one-off. I haven't found a better solution to this apart from just re-run the workflow.

File: cron/k8s/transfer.release-raw.yaml (hunk 1 above)
Comment (laurentsimon) [reply cron/k8s/transfer.release-raw.yaml:36]: I like yours better, but the other table is called `scorecard_releasedataset` (which is why I named it this way). More than happy to update, though

Comment (azeemshaikh38) [reply cron/k8s/transfer.release-raw.yaml:36]: Upto you. If it's not a lot of work to change the table name, can use the existing convention.

**Pull request merged**